EIPS/eip-7693.md

```diff
@@ -0,0 +1,241 @@
+---
+eip: 7693
+title: Backward-Compatible Post-Quantum Migration
+description: This EIP aims to bring quantum resistance to Ethereum through a novel backward-compatible approach.
```

Comment:
```suggestion
description: Make accounts quantum resistant by adding a ZK proof of knowledge to each signature.
```

We can omit the fluff about this EIP, since this description obviously applies to this EIP. It's also somewhat redundant to mention Ethereum, since all ERCs/EIPs are Ethereum related. It's also quite useful if the description gives a bit more detail about the internals of the proposal, instead of just being a teaser.